SofaKernel/modules/SofaMeshCollision/src/SofaMeshCollision/TriangleModel.inl

```diff
@@ -19,32 +19,20 @@
 *                                                                             *
 * Contact information: contact@sofa-framework.org                             *
 ******************************************************************************/
-//#ifndef SOFA_COMPONENT_COLLISION_TRIANGLEMODEL_INL
-//#define SOFA_COMPONENT_COLLISION_TRIANGLEMODEL_INL
-
+//#pragma once
```

Comment:
Why commented?

Reply:
because my perl script replace the #ifndef blabla but was not supposed to know it was commented :F